docs/content/installation/csi_ug_requirements.md

```diff
@@ -37,33 +37,50 @@ Perform these steps for each worker node in Kubernetes cluster to prepare your e
 
-   Download and save the following yaml file:
+   Download and save the following YAML file:
 
    ```
    curl https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/master/deploy/99-ibm-attach.yaml > 99-ibm-attach.yaml
    ```
 
    This file can be used for both Fibre Channel and iSCSI configurations. To support iSCSI, uncomment the last two lines in the file.
 
-   **Important:** The 99-ibm-attach.yaml configuration file overrides any files that already exist on your system. Only use this file if the files mentioned are not already created. <br />If one or more have been created, edit this yaml file, as necessary.
+   **Important:** The `99-ibm-attach.yaml` configuration file overrides any files that exist on your system. Only use this file if the files mentioned are not already created. <br />If one or more were created, edit this YAML file, as necessary.
 
-   Apply the yaml file.
+   Apply the YAML file.
 
    `oc apply -f 99-ibm-attach.yaml`
-    
-3. If needed, enable support for volume snapshots (FlashCopy® function) on your Kubernetes cluster.
 
-   For more information and instructions, see the Kubernetes blog post, [Kubernetes 1.20: Kubernetes Volume Snapshot Moves to GA](https://kubernetes.io/blog/2020/12/10/kubernetes-1.20-volume-snapshot-moves-to-ga/).
+3. Configure storage system connectivity.
 
-   Install both the Snapshot CRDs and the Common Snapshot Controller once per cluster.
+    1.  Define the host of each Kubernetes node on the relevant storage systems with the valid WWPN (for Fibre Channel) or IQN (for iSCSI) of the node.
 
-   The instructions and relevant yaml files to enable volume snapshots can be found at: [https://github.com/kubernetes-csi/external-snapshotter#usage](https://github.com/kubernetes-csi/external-snapshotter#usage)
+    2.  For Fibre Channel, configure the relevant zoning from the storage to the host.
+    
+4. (Optional) If planning on using volume snapshots (FlashCopy® function), enable support on your Kubernetes cluster.
 
-4. Configure storage system connectivity.
+   For more information and instructions, see the Kubernetes blog post, [Kubernetes 1.20: Kubernetes Volume Snapshot Moves to GA](https://kubernetes.io/blog/2020/12/10/kubernetes-1.20-volume-snapshot-moves-to-ga/).
 
-    1.  Define the host of each Kubernetes node on the relevant storage systems with the valid WWPN (for Fibre Channel) or IQN (for iSCSI) of the node.
+   Install both the Snapshot CRDs and the Common Snapshot Controller once per cluster.
 
-    2.  For Fibre Channel, configure the relevant zoning from the storage to the host.
+   The instructions and relevant YAML files to enable volume snapshots can be found at: [https://github.com/kubernetes-csi/external-snapshotter#usage](https://github.com/kubernetes-csi/external-snapshotter#usage)
 
+5. (Optional) If planning on using volume replication (remote copy function), enable support on your Kubernetes cluster and storage system.
```

Comment:
note that this is also true for ocp clusters (not only kubernetes)

Reply:
DONE

Reply:
same for [`Kubernetes replication objects`](https://github.com/IBM/ibm-block-csi-driver/blob/8d18ae5a6d95b15894c814e59a27c133aea06d8a/docs/content/release_notes/csi_rn_limitations.md#volume-replication-limitations) -> `orchestration platform replication objects`

Reply:
DONE

Reply:
updating. Next time please try to be a bit more specific as to what needs to be updated/where or comment on that specific file section. 
DONE

Reply:
I'm not familiar with an easy way to comment on all specific locations, the best way I can recommend for you is Ctrl+F in the `Files changed` tab

Reply:
I understand but if you can just specify / write it out in the comment. otherwise sometimes it takes me a while to find it

Reply:
it takes me a while to find it as well, I think it's best to avoid relying on many cumbersome links